README.adoc

```diff
@@ -88,51 +88,15 @@ Once you're done checking out the application, log out and stop the Open Liberty
 mvn liberty:stop-server
 ```
 
-// =================================================================================================
-//
-// =================================================================================================
-== Configuring the user registry
-
-Navigate to the  `start` directory to begin.
-
-You need to set up the user registry to define users and role information. User registries store user account information, such as user name and password, that can be accessed during authentication.
-Typically, applications would use an external and configurable registry, like a database or Lightweight Directory Access Protocol (LDAP). However, for this guide, a basic registry provided by Liberty is used as a simple example of an user registry.
-
-Create the `src/main/liberty/config/userRegistry.xml` file:
-
-[source, xml, indent=0]
-----
-include::finish/src/main/liberty/config/userRegistry.xml[tags=**]
-----
-
-On the Open Liberty server, you can use the basic registry as an user registry to define the users and groups information.
-In this application, there are four groups: `Employee`, `TeamLead`, `Manager` and `PartTime`. Groups can be assigned to different access roles.
-
-Open the `server.xml` file to view the configuration that maps roles to user groups.
-
-[source, xml, indent=0]
-----
-include::finish/src/main/liberty/config/server.xml[tags=security]
-----
-
-Within the `<application-bnd>` tags, two different authorization roles are assigned to these groups: `admin` and `user`.
-
-A role can be mapped to a user, a group, or a special subject. The two types of special subject are `EVERYONE` and `ALL_AUTHENTICATED_USERS`.
-When a role is mapped to the `EVERYONE` special subject, there's no security because everyone is allowed access and you're not prompted to enter credentials.
-When a role is mapped to the `ALL_AUTHENTICATED_USERS` special subject, then any user who is authenticated by the application server can access the protected resource.
-
-=== Encrypting passwords using securityUtility
-
-It is not recommended to store password in plain text. For simplicity reasons, the application in this guide uses XOR encoding to encrypt the passwords, as you can see in the `userRegistry.xml`. For example, to encrypt 'bobpwd', navigate to the `target/liberty/wlp/bin` directory and run the Liberty `securityUtility` command:
-```
-./securityUtility encode --encoding=xor bobpwd
-
-```
 // =================================================================================================
 //
 // =================================================================================================
 
 == Adding authentication and authorization
+First, let's build and secure this application. Navigate to the  `start` directory to begin.
+
+This application consists of a home servlet that redirects the user to the login page where the user's credentials are authenticated.
```

Comment:
I wonder here you want to use `HomeServlet` instead of "home servlet" to make it consistent in this section. The reason being, `HomeServlet` class name is self explanatory enough. Not a problem tho.

Reply:
I will leave it as is right now and see if ID or others have suggestions for this point.

Reply:
It looks ok for now.